src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -206,11 +182,17 @@ private bool TryGetTextForSymbol(SyntaxToken token, SemanticModel semanticModel,
                 return false;
             }
 
-            text = symbol != null ? FormatSymbol(symbol) : null;
-            return symbol != null;
+            if (symbol is IDiscardSymbol)
+            {
+                text = Keyword("_");
```

Comment:
> I would prefer Keyword("discard") to make it clearer, and avoid future disambiguation on the docs side.

@BillWagner what do you think?  I can see that we send punction for a lot of the symbols (like `?_CSharpKeyword`).  Would you prefer us to continue that here, os use a name like `discard`?

Reply:
> Also, how do you feel about handling IParameterSymbol where the name is _ because _ => goo(); is not a discard to the compiler, but probably is to the user

I'll think about it, but i don't feel great about it :)

Reply:
We certainly do send punctuation in other scenarios, but then also have to come in here and disambiguate later (eg https://github.com/dotnet/roslyn/pull/47516 and https://github.com/dotnet/roslyn/pull/46987). Feels like this is a good opportunity to preempt that.. for example, we should probably add help for the digit separator, which is also an underscore.

Reply:
k!